dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE9Action.java

```diff
@@ -0,0 +1,226 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package componenttest.rules.repeater;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import com.ibm.websphere.simplicity.log.Log;
+import com.ibm.ws.jakarta.transformer.JakartaTransformer;
+
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.custom.junit.runner.RepeatTestFilter;
+
+/**
+ * Test repeat action that will do 2 things:
+ * <ol>
+ * <li>Invoke the Jakarta transformer on all war/ear files under the autoFVT/publish/ folder</li>
+ * <li>Update all server.xml configs under the autoFVT/publish/ folder to use EE 9 features</li>
+ * </ol>
+ */
+public class JakartaEE9Action extends FeatureReplacementAction {
+
+    private static final Class<?> c = JakartaEE9Action.class;
+
+    public static final String ID = "EE9_FEATURES";
+
+    static final String[] EE9_FEATURES_ARRAY = { "jakartaee-9.0", "webProfile-9.0", "jakartaeeClient-9.0", "servlet-5.0", "jdbc-4.2", "javaMail-2.0", "cdi-3.0", "jpa-3.0",
```

Comment:
Why is jdbc-4.2 an EE9 feature?  Also, concurrent-2.0 is missing.  Are others missing as well - I don't see EJB (whatever the new name for it is) ?

Reply:
jdbc-4.2 doesn't have to be in here, what this implies is that when server.xml's are getting upgraded to EE9-level features, we will also upgrade to jdbc-4.2 if older jdbc version are specified like 4.0 or 4.1. While tests would technically work with any JDBC feature version, I think it would be good for us to test with the latest features when possible -- this list can also be customized at the FAT bucket level, so if a FAT wants to specifically test an older JDBC version it can still do so.

Regarding other features like concurrent-2.0, yes I will get those added.